<ui version="4.0" >
 <class>MSMSWin</class>
 <widget class="QDialog" name="MSMSWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1180</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MS/MS View</string>
  </property>
  <layout class="QHBoxLayout" >
   <item>
    <widget class="QGraphicsView" name="graphicsView" />
   </item>
   <item>
    <widget class="QToolBox" name="toolBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Fixed" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize" >
      <size>
       <width>320</width>
       <height>16</height>
      </size>
     </property>
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="page" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>176</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>History</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QListWidget" name="historyList" />
       </item>
       <item>
        <widget class="QPushButton" name="clearHistoryButton" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>176</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Axes</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>max intensity</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="maxIEdit" />
       </item>
       <item row="1" column="0" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>max mz</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2" >
        <widget class="QLineEdit" name="maxmzEdit" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>min mz</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2" >
        <widget class="QLineEdit" name="minmzEdit" />
       </item>
       <item row="3" column="0" colspan="2" >
        <widget class="QPushButton" name="resetAxesButton" >
         <property name="text" >
          <string>Reset Axes</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" >
        <widget class="QPushButton" name="updateAxesButton" >
         <property name="text" >
          <string>Update</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>320</width>
        <height>176</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>Data</string>
      </attribute>
      <widget class="QLabel" name="chargeLabel" >
       <property name="geometry" >
        <rect>
         <x>10</x>
         <y>10</y>
         <width>172</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text" >
        <string>Precursor Charge: +1</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>112</width>
        <height>112</height>
       </rect>
      </property>
      <attribute name="label" >
       <string>DB Search</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QListWidget" name="fragmentList" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
